src/scss/styles.scss

```diff
@@ -37,11 +37,12 @@
 @import "node_modules/@department-of-veterans-affairs/formation/sass/base/b-mixins";
 @import "node_modules/@department-of-veterans-affairs/formation/sass/modules/m-dropdown";
 @import "node_modules/@department-of-veterans-affairs/formation/sass/modules/m-form-elements";
```

Comment:
Did you mean to leave this in? There's a local m-form-elements now.

Reply:
Yes, I meant to leave it in. Unfortunately, not all of the styles in the m-form-elements file from the design-system could be moved here, so I pulled out only the ones that can be for now.

Reply:
Ah, ok. It may turn out we want to refactor some of that into its own components if they're usable outside this repo by themselves.

Reply:
Yes, absolutely!